components/templates/src/builtins/shortcodes/twitch.html

```diff
@@ -0,0 +1,9 @@
+<div {% if class %}class="{{class}}"{% endif %}>
+    <iframe
+        src="https://player.twitch.tv/?channel={{channel}}" 
```

Comment:
it's possible to embed chat etc according to https://dev.twitch.tv/docs/v3/guides/embed-chat so maybe call this one `twitch_channel`?

https://discuss.dev.twitch.tv/t/can-you-embed-twitchs-chat-player-to-a-streamers-own-website-that-has-minimal-ads-sponsor-spots-live-samples-in-post/7380/2

Reply:
Or maybe one shortcode that takes a param `chat=bool` to keep things simple and will either render the video or the chat

Reply:
We can have 2 shortcodes, one for the chat and one for the stream (twitch_stream and twitch_chat), so they can be added independently to a page. What do you think @Keats?

I'll check it in a couple of hours, also I will add the docs.

Reply:
I think a single shortcode is probably better with the following default: `twitch(channel="SOMETHING", chat=false)`. By default if called `twitch(channel="hey")` it will load the video but you can get the chat by doing `twitch(channel="hey", chat=true)`.

I don't think it's worth having 2 shortcodes for that